### ISSUE TITLE
Defining *** Test Case / Task *** section in resource file is reported as parsing-error

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#525]
Related #524 

If Robocop auto detect file as resource type, but it will contain `*** Test Case / Task ***` section it will be reported as parsing error. It should be parsed and reported as other type of issue:
test.robot:
```
*** Settings ***
Resource    resource.robot

```
resource.robot:
```
*** Test Cases ***
Test
    No Operation

```
Command:
```
robocop test.robot resource.robot
```
Output:
```
resource.robot:1:1 [E] 0401 Robot Framework syntax error: Resource file with 'Test Cases' section is invalid. (parsing-error)
```

**Comment by mnojek** [comment]
@rikerfi FYI New rule E0415 `invalid-section-in-resource` is released in latest [Robocop 3.1.0](https://github.com/MarketSquare/robotframework-robocop/blob/master/docs/releasenotes/3.1.0.rst) :)

You can install the latest available version by:
```
pip install --upgrade robotframework-robocop
```
